static/src/stylesheets/module/commercial/gustyle/_gu-comcontent.scss

```diff
@@ -35,6 +35,7 @@
     color: #ffffff;
 }
 
+.content--immersive .from-content-api .gu-display__content-title,
```

Comment:
It may well be, although it's not the real fix for this problem in the long term.

The long term fix is to try and disentangle our CSS and give this commercial content a 'blank slate' to work upon.